test/BinaryToText.cpp

```diff
@@ -285,6 +288,8 @@ INSTANTIATE_TEST_CASE_P(LoopControlMasks, RoundTripInstructionsTest,
                             "OpLoopMerge %1 %2 Unroll\n",
                             "OpLoopMerge %1 %2 DontUnroll\n",
                             "OpLoopMerge %1 %2 Unroll|DontUnroll\n",
+                            "OpLoopMerge %1 %2 DependencyInfinite\n",
+                            "OpLoopMerge %1 %2 DependencyLength 8\n",
```

Comment:
Perhaps this should go into `TextToBinary.ControlFlow.cpp`, together with other `OpLoopMerge` tests?

See 13877a3b0e9d9e3217f726178f6d26f1bf1867cc.